package.json

```diff
@@ -84,7 +84,7 @@
   },
   "dependencies": {
     "@artsy/backbone-mixins": "3.0.0",
-    "@artsy/cohesion": "4.40.0",
+    "@artsy/cohesion": "^4.40.0",
```

Comment:
Hm, how'd this unrelated package update get in here? I'm assuming we want this dependency pinned.

Reply:
I just wrote a comment about that @dzucconi . I think its because I ran `yarn add @artsy/cohesion --latest` to get access to the `clickedArtworkGroup` event on this branch?